contracts/SportMarkets/SportsAMM.sol

```diff
@@ -683,7 +683,8 @@ contract SportsAMM is Initializable, ProxyOwned, PausableUpgradeable, ProxyReent
         address _referrals,
         address _parlayAMM,
         address _wrapper,
-        address _lp
+        address _lp,
+        address _riskManager
```

Comment:
Not a fan of extending this function due to pDAO upgrades, but let's stop with the risk manager, as a last one. 
It's 9 attempts to make a copy-paste mistake

Reply:
true